src/modules/emotes/channel-emotes.js

```diff
@@ -15,40 +15,42 @@ const category = {
 class ChannelEmotes extends AbstractEmotes {
   constructor() {
     super();
-
+    this._updatedChannel = null;
```

Comment:
instead of storing this on the class, can we store this outside of the class because this is a singleton? also can we rename this to `currentChannel`